CHANGELOG.asciidoc

```diff
@@ -165,7 +167,7 @@ New release with various improvements and bugfixes:
 * https://github.com/devonfw/ide/issues/692[#692]: Improve Windows Terminal with gitbash
 * https://github.com/devonfw/ide/issues/786[#786]: Improve Windows Terminal with powershell
 * https://github.com/devonfw/ide/issues/849[#849]: Integration of GCViewer
-* https://github.com/devonfw/ide/issues/867[#867]: ERROR: Error opening the file. There may be a disk or file system error.
+* https://github.com/devonfw/ide/issues/867[#867]: ERROR: Error opening the file, there may be a disk or file system error
```

Comment:
are you using auto-magic AsciiDoc plugins that try to do "auto-correction".
This is the summary of an issue and it comes from an actually error message that is just the way this error message is.
Chaning this title here does not seem to make sense.
Tame your IDE :)

Reply:
I've tried to manually fix the conflict between end of lines with a dot and a required newline.
It's reverted now.